partner_statement/report/__init__.py

```diff
@@ -5,3 +5,5 @@
 from . import activity_statement_xlsx
 from . import detailed_activity_statement_xlsx
 from . import outstanting_statement_xlsx
+from . import current_statement
+from . import current_statement_xlsx
```

Comment:
Is there too much difference between current_statement and outstanding_statement? If not, perhaps it's better to make current_statement be an inherit of outstanding_statement, the same way detailed_activity_statement inherits activity_statement. Surely it would reduce code.


Or maybe it should just merged into the outstanding_statement by adding an extra boolean parameter.